docs/classes-emit.md

```diff
@@ -54,15 +54,15 @@ Here `d` refers to the derived class and `b` refers to the base class. This func
 1. copies the static members of the base class onto the child class i.e. `for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];`
 1. sets up the child class function's prototype to optionally lookup members on the parent's `proto` i.e. effectively `d.prototype.__proto__ = b.prototype`
 
-People rarely have trouble understanding 1, but many people struggle with 2. so an explanation is in order
+People rarely have trouble understanding 1, but many people struggle with 2. So an explanation is in order.
 
 #### `d.prototype.__proto__ = b.prototype`
 
 After having tutored many people about this I find the following explanation to be simplest. First we will explain how the code from `__extends` is equivalent to the simple `d.prototype.__proto__ = b.prototype`, and then why this line in itself is significant. To understand all this you need to know these things:
 1. `__proto__`
-1. `prototype`
-1. effect of `new` on `this` inside the called function
-1. effect of `new` on `prototype` and `__proto__`
+2. `prototype`
+3. effect of `new` on `this` inside the called function
+4. effect of `new` on `prototype` and `__proto__`
 
```

Comment:
Yes, numbers get properly updated on the website (https://basarat.gitbooks.io/typescript/content/docs/classes-emit.html) and in .pdf version, but looks like github preview doesn't update numbers - please see the screenshot

<img width="1095" alt="numbered-list" src="https://cloud.githubusercontent.com/assets/534603/21133226/90605d2e-c120-11e6-953d-d1acee8dd74b.png">
(https://github.com/basarat/typescript-book/blob/master/docs/classes-emit.md). So, explicit numbering with 1,2,3,4 can be more universal solution. What do you think about it @basarat ?